utils/scripts/helpers/repo-helper-el9.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -uex
+
+# This script is used by dockerfiles to optionally use
+# a local repository instead of a distro provided repository.
+# It will also optionally allow running a /tmp/install script
+# for custom packages if present.
+
+: "${REPO_FILE_URL:=}"
+: "${BASE_DISTRO:=rockylinux/rockylinux:$MAJOR_VER}"
+: "${JENKINS_URL:=}"
+: "${REPOS:=}"
+
+# shellcheck disable=SC2120
+disable_repos () {
+    local repos_dir="$1"
+    shift
+    local save_repos
+    IFS=" " read -r -a save_repos <<< "${*:-} daos_ci-el$MAJOR_VER-artifactory"
+    if [ -n "$REPO_FILE_URL" ]; then
+        pushd "$repos_dir"
+        local repo
+        for repo in "${save_repos[@]}"; do
+            mv "$repo".repo{,.tmp}
+        done
+        for file in *.repo; do
+            true > "$file"
+        done
+        for repo in "${save_repos[@]}"; do
+            mv "$repo".repo{.tmp,}
+        done
+        popd
+    fi
+}
+
+# Use local repo server if present
+install_curl() {
+    :
+}
```

Comment:
What does this do?

Reply:
It's an empty stub for a function that doesn't need to do anything on this platform.  It helps in maintaining the `repo-helper-*.sh` scripts so that one can easily see what the difference is between them when viewed in a side-by-side diff with, I.e. `gvim -d utils/scripts/helpers/repo-helper-*.sh`.